plugins/data-types/src/utils/unitCheck.ts

```diff
@@ -22,7 +22,12 @@ export const unitCheck = (series: TypedSeries): void => {
 	}
 
 	const unit = test[1]
-	if (!unit || ignored.includes(unit)) return
+	if (
+		!unit ||
+		ignored.includes(unit.toUpperCase()) ||
+		ignored.includes(series.name.toUpperCase())
```

Comment:
What does this new condition do? Why do we check series name against ignored (not a unit) list?